### ISSUE TITLE
E_VALIDATION error trying to update a record containing a "time" field

### ISSUE THREAD

**Issue by kpturner** [balderdashy/sails-mysql#241]
I get a validation error whenever I try to update a record containing a "time" field (as opposed to "date" or "datetime").

I can create a record manually using PHPMYADMIN with a time of, for example, "13:00:00".   I can retrieve it using a find method and the resulting JSON shows the field with this exact value.  If I then try to update the record (without changing any data), the update throws an error:

{"error":"E_VALIDATION","status":400,"summary":"1 attribute is invalid","model":"Event","invalidAttributes":{"time":[{"message":"[object Object]"}]}}

Is it a bug, or am I missing something fundamental?

**Comment by sailsbot** [comment]
Thanks for posting, @kpturner.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
